include/tc/core/cuda/cuda_tc_executor.h

```diff
@@ -35,15 +35,10 @@ class CudaTcExecutor : public ::tc::TcExecutor {
   CudaTcExecutor(
       std::string id,
       std::vector<const DLTensor*> inputsInfo,
-      const MappingOptions& options,
+      const std::string& options,
```

Comment:
These changes can be squashed into c91c2c6c81e223aea93a829eb504dd62dc8681c1 to resolve
@ftynse's remark.